coffee-chats/src/main/webapp/src/components/NavBar.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+
+import {AppBar, Grid, Icon, IconButton, Toolbar, Tooltip} from "@material-ui/core";
+
+export function NavBar() {
+  return (
+      <AppBar position="static" style={{ background: "transparent", boxShadow: "none"}}>
+        <Toolbar>
+          <Grid justify="space-between" container>
+            <Grid item>
+              {/* Left side of the navbar */}
```

Comment:
`<Grid container justify="flex-end"`> to avoid need of empty elements and complicated nested structure of containers?

Reply:
Yeah, I was just planning on adding a hamburger button on the left anyways (see groups-page-prototype branch)